src/components/Navbar.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { observer } from "mobx-react"
+import swal from 'sweetalert';
```

Comment:
Please explain your changes here.

Reply:
We are using swal in the nav bar for diff check popup and HTML export... We have import before we use it and I'm not sure how it's working in our old app.